### DISCUSSION TITLE
Populate X-Consumer-Groups without ACL plugin

### DISCUSSION THREAD

**Discussion by FabienForestier** [Kong/kong#7325] [Ideas and feature requests]
# Summary

It would be a nice feature to be able to populate the X-Consumer-Groups headers without having to add the ACL plugin.

Indeed in my case, the authorization is handled in my upstream services partly based on Kong groups. In order to have the groups header I need to activate the ACL plugin globally even if I don't want to deny any groups. My turnaround is denying a none existing group (which was working in version 1.1.1 of kong but seems not to work in the newer version) but it would be better to have a proper way to do that.

**Comment by bungle** [comment]
I think this is a good idea.

**Comment by bungle** [comment]
Also `denying none` groups feels a bug to me. Can you give us a quick way to reproduce.

**Comment by FabienForestier** [comment]
I think I didn't expressed my self clearly. In the deny input, I enter a group name that I am sure will never exists in my platform. It's a string made of random characters.

**Comment by bungle** [comment]
@FabienForestier, yes I understood that.
